src/test/java/seedu/duke/ParserTest.java

```diff
@@ -39,7 +39,7 @@ public void parse_emptyString_exceptionThrown() {
             defaultParser.parse(fullCommand);
         });
 
-        assertEquals(e.getMessage(), Constants.EXCEPTION_PARSER_EMPTYSTRING);
+        assertEquals(e.getMessage(), Constants.INVALID_INPUT_EMPTY_STRING);
```

Comment:
The order of the arguments for `assertEquals` is `(expected, actual)`, so it would be good to swap them around for consistency.

Reply:
Thank you! Corrections has been pushed.